devel/idevicerestore/Portfile

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        libimobiledevice idevicerestore 1.0.0
+
+categories          devel
+platforms           darwin
+
+license             LGPL-3
+maintainers         {@i0ntempest me.com:szf1234} openmaintainer
+
+description         The libirecovery library allows communication with iBoot/iBSS of iOS devices via USB.
+
+long_description    The idevicerestore application is a full reimplementation of all granular steps which \
+                    are performed during the restore of a firmware to a device. In general, upgrades and \
+                    downgrades are possible, however subject to availability of SHSH blobs from Apple for \
+                    signing the firmare files. Some key features are:\n\
+                    * Restore: Update firmware on iOS devices\n\
+                    * Firmware: Use official IPSW firmware archive file or a directory as source\n\
+                    * Update: Allows updating the device by default or erasing all data\n\
+                    * Download: On demand download of latest available firmware for a device\n\
+                    * Cache: Downloaded firmware files are cached locally\n\
+                    * Custom Firmware: Restore custom firmware files (requires bootrom exploit)\n\
+                    * Baseband: Allows you to skip NOR/Baseband upgrade\n\
+                    * SHSH: Fetch TSS records and save them as ".shsh" files\n\
+                    * DFU: Put devices in pwned DFU mode (limera1n devices only)\n\
+                    * AP Ticket: Use custom AP ticket from a file\n\
+                    * Cross-Platform: Tested on Linux, macOS, Windows and Android platforms\n\
+                    * History: Developed since 2010
+
+homepage            https://www.libimobiledevice.org/
+
+checksums           rmd160  f774ac598099e965bf4e9dcad53078e6dab66479 \
+                    sha256  76ee2570745f9e876f89acf82ad91edf6e9083e08ceea429985a480cae1ddd67 \
+                    size    106889
+
+if {${subport} eq ${name}} {
+    patchfiles          patch-postrelease-fixes.diff
+}
+
+depends_build-append \
+                    port:autoconf \
+                    port:automake \
+                    port:libtool \
+                    port:pkgconfig
+
+depends_lib         port:curl \
+                    port:libusb \
+                    port:libimobiledevice \
+                    port:libirecovery \
+                    port:libplist \
+                    port:libzip \
+                    port:openssl \
```

Comment:
Unless you know for a fact that this port will not work with libressl, you should allow it to be used by writing this dependency in `path:` style like most other ports that use openssl.